ui/static/ui/js/utils.jsx

```diff
@@ -32,23 +32,25 @@ define("utils", ["jquery", "lodash", "reactaddons", "select2"],
    */
   var StatusBox = React.createClass({
     render: function() {
-      if (!this.props || !this.props.message) {
+      if (!this.props) {
         return null;
       }
 
-      if (this.props.message.error !== undefined) {
-        return <div className="alert alert-danger alert-dismissible">
+      if (this.props.message && this.props.message.error !== undefined) {
+        return <div
+          className="alert alert-danger alert-dismissible status-container">
```

Comment:
Since the only reason you're defining `status-container` is to get its position, and you can just scroll the panel to the top it seems like you don't need these changes anymore